addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py

```diff
@@ -55,6 +56,14 @@ def gather_material(blender_material, mesh_double_sided, export_settings):
         pbr_metallic_roughness=__gather_pbr_metallic_roughness(blender_material, orm_texture, export_settings)
     )
 
+    user_extensions = export_user_extensions(material, blender_material, export_settings)
+    if user_extensions is not None:
```

Comment:
I don't see why you need to have `export_user_extensions` return anything. Just let the user-extension code add anything it wants to `material.extensions`.